deploy/overlays/dev-template/system-config.yaml

```diff
@@ -10,3 +10,7 @@ data:
   builder-image.jdk8.image: jdk8-builder
   builder-image.jdk11.image: jdk11-builder
   builder-image.jdk17.image: jdk17-builder
+  # pulled the tags from https://github.com/redhat-appstudio/infra-deployments/blob/pre-kcp/components/build/jvm-build-service/system-config.yaml#L13-L15
+  builder-image.jdk8.tags: jdk:8,maven:3.8,gradle:7.4.2;6.9.2;5.6.4;4.10.3
```

Comment:
These should be coming from https://github.com/stuartwdouglas/jvm-build-service/blob/4a78dab7188f16f1a175de169f55590cb4236508/deploy/base/system-config.yaml#L11 , and once the new builder repo is live it will reference the config file at https://github.com/redhat-appstudio/jvm-build-service-builder-images/blob/main/image-config.yaml

Why is this needed here?

Reply:
ah cause I didn't realize that other copy was there

and sure, now that I see it there, I get where the kustomize merge comes in

and I'm still use to having *one* @#$%@#$% copy of a api obj yaml file :-)

thanks will remove